### ISSUE TITLE
Statistics not updating after upgrade to 24.3 - Clickhouse Exception: Net Exception: Socket is not connected

### ISSUE THREAD

**Issue by optixx** [getsentry/self-hosted#2915]
### Self-Hosted Version

24.3

### CPU Architecture

x86_64

### Docker Version

25.0.3

### Docker Compose Version

2.24.6

### Steps to Reproduce

1. Upgrade from `24.2` to `24.3` using `install.sh`
2. Process error messages

### Expected Result

Get updated statistics for the processed errors in web ui

### Actual Result

We see no updated statistics in the web ui

Looks like a clickhouse problem:
```
sentry-self-hosted-clickhouse-1                 | Processing configuration file '/etc/clickhouse-server/config.xml'.
sentry-self-hosted-clickhouse-1                 | Merging configuration file '/etc/clickhouse-server/config.d/docker_related_config.xml'.
sentry-self-hosted-clickhouse-1                 | Merging configuration file '/etc/clickhouse-server/config.d/sentry.xml'.
sentry-self-hosted-clickhouse-1                 | Logging warning to /var/log/clickhouse-server/clickhouse-server.log
sentry-self-hosted-clickhouse-1                 | Logging errors to /var/log/clickhouse-server/clickhouse-server.err.log
sentry-self-hosted-clickhouse-1                 | Logging warning to console
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:31:32.425727 [ 44 ] {} <Warning> Application: Listen [::]:8123 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.8.13.1.altinitystable (altinity build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:31:32.426056 [ 44 ] {} <Warning> Application: Listen [::]:9000 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.8.13.1.altinitystable (altinity build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:31:32.426174 [ 44 ] {} <Warning> Application: Listen [::]:9009 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.8.13.1.altinitystable (altinity build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:31:32.426291 [ 44 ] {} <Warning> Application: Listen [::]:9004 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.8.13.1.altinitystable (altinity build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:31:32.426431 [ 44 ] {} <Warning> Application: Listen [::]:9005 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.8.13.1.altinitystable (altinity build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
sentry-self-hosted-clickhouse-1                 | 2024.03.21 11:33:01.035477 [ 47 ] {} <Error> ServerErrorHandler: Poco::Exception. Code: 1000, e.code() = 107, e.displayText() = Net Exception: Socket is not connected, Stack trace (when copying this message, always include the lines below):
sentry-self-hosted-clickhouse-1                 |
sentry-self-hosted-clickhouse-1                 | 0. Poco::Net::SocketImpl::error(int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) @ 0x13c4ee8e in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 1. Poco::Net::SocketImpl::peerAddress() @ 0x13c510d6 in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 2. DB::HTTPServerRequest::HTTPServerRequest(std::__1::shared_ptr<DB::Context const>, DB::HTTPServerResponse&, Poco::Net::HTTPServerSession&) @ 0x110e6f0b in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 3. DB::HTTPServerConnection::run() @ 0x110e5d6e in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 4. Poco::Net::TCPServerConnection::start() @ 0x13c5614f in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 5. Poco::Net::TCPServerDispatcher::run() @ 0x13c57bda in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 6. Poco::PooledThread::run() @ 0x13d89e59 in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 7. Poco::ThreadImpl::runnableEntry(void*) @ 0x13d860ea in /usr/bin/clickhouse
sentry-self-hosted-clickhouse-1                 | 8. start_thread @ 0x9609 in /usr/lib/x86_64-linux-gnu/libpthread-2.31.so
sentry-self-hosted-clickhouse-1                 | 9. clone @ 0x122293 in /usr/lib/x86_64-linux-gnu/libc-2.31.so
sentry-self-hosted-clickhouse-1                 |  (version 21.8.13.1.altinitystable (altinity build))

```

**Comment by jast** [comment]
The error message was a red herring, but we fixed the stats problem by applying #2909. Thanks!

**Comment by aldy505** [comment]
Since you resolved the stats error by applying that fix, I'm going to close this issue since the Clickhouse error problem is already stated here: https://github.com/getsentry/self-hosted/issues/2876

Feel free to add "me too!!!" just for the fun of it :laughing:
